cloudflare/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudflare"
-version = "0.9.1"
```

Comment:
We don't bump this until we do a release PR

Reply:
You know what, this procedure can probably change. I think that bumping it to a 0.X+1 alpha in the event of a breaking change is probably a better system overall.